k8s/hazelcast/apptest/deployer/hazelcast/values.yaml

```diff
@@ -0,0 +1 @@
+testerImage: null
```

Comment:
Add a tester image

Reply:
schema.yaml adds for us. Check other apps, please:

1. https://github.com/GoogleCloudPlatform/click-to-deploy/blob/master/k8s/activemq/apptest/deployer/activemq/values.yaml

2. https://github.com/GoogleCloudPlatform/click-to-deploy/blob/master/k8s/activemq/apptest/deployer/schema.yaml